src/Behat/Behat/Context/Snippet/Generator/ContextSnippetGenerator.php

```diff
@@ -317,43 +262,30 @@ private function getMethodNameNotProposedEarlier($contextClass, $stepPattern, $n
 
     /**
      * Returns already proposed method names.
-     *
-     * @param string $contextClass
-     *
-     * @return string[]
      */
-    private function getAlreadyProposedMethods($contextClass)
+    private function getAlreadyProposedMethods(string $contextClass): array
     {
-        return self::$proposedMethods[$contextClass] ?? array();
+        return self::$proposedMethods[$contextClass] ?? [];
     }
 
     /**
      * Marks method as proposed one.
-     *
-     * @param string $contextClass
-     * @param string $stepPattern
-     * @param string $methodName
      */
-    private function markMethodAsAlreadyProposed($contextClass, $stepPattern, $methodName)
+    private function markMethodAsAlreadyProposed(string $contextClass, string $stepPattern, string $methodName): void
     {
         self::$proposedMethods[$contextClass][$stepPattern] = $methodName;
     }
 
     /**
      * Returns method argument.
-     *
-     * @param string $argument
-     *
-     * @return string
      */
-    private function getMethodArgument($argument)
+    private function getMethodArgument(mixed $argument): string
     {
-        $arg = '__unknown__';
-        if ($argument instanceof PyStringNode) {
-            $arg = 'PyStringNode $string';
-        } elseif ($argument instanceof TableNode) {
-            $arg = 'TableNode $table';
-        }
+        $arg = match (true) {
```

Comment:
You can get rid of this temporary variable and just return the result of the match